examples/notebook.ipynb

```diff
@@ -87,14 +107,62 @@
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "MotionGroupStateResponse(state=MotionGroupState(motion_group='0@ur', controller='ur', joint_position=Joints(joints=[1.1699999570846558, -1.6579999923706055, 1.4049999713897705, -1.5709999799728394, -1.5709999799728394, 1.1699999570846558]), joint_velocity=Joints(joints=[0, 0, 0, 0, 0, 0]), joint_torque=None, flange_pose=Pose(position=Vector3d(x=-91.40618174243164, y=-662.0208072831879, z=851.325747432342), orientation=Vector3d(x=2.1397304301105744, y=2.1396749657965426, z=-0.35713579892606034), coordinate_system=''), tcp_pose=TcpPose(position=Vector3d(x=-91.40618174243164, y=-662.0208072831879, z=851.325747432342), orientation=Vector3d(x=2.1397304301105744, y=2.1396749657965426, z=-0.35713579892606034), coordinate_system='', tcp='Flange'), velocity=MotionVector(linear=Vector3d(x=0, y=0, z=0), angular=Vector3d(x=0, y=0, z=0), coordinate_system=''), force=ForceVector(force=Vector3d(x=0, y=0, z=0), moment=Vector3d(x=0, y=0, z=0), coordinate_system=''), joint_limit_reached=MotionGroupStateJointLimitReached(limit_reached=[False, False, False, False, False, False]), joint_current=Joints(joints=[0, 0, 0, 0, 0, 0])), tcp_pose=TcpPose(position=Vector3d(x=-91.40618174243164, y=-662.0208072831879, z=851.325747432342), orientation=Vector3d(x=2.1397304301105744, y=2.1396749657965426, z=-0.35713579892606034), coordinate_system='', tcp='Flange'))"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "state = await motion_group.get_state(\"Flange\")\n",
+    "state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "d0d6a325",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m2024-12-19 12:57:15.041\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m31\u001b[0m - \u001b[1mCalling get_current_motion_group_state with args=(), kwargs={'cell': 'cell', 'motion_group': '0@ur', 'tcp': 'Flange'}\u001b[0m\n",
+      "\u001b[32m2024-12-19 12:57:15.139\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m37\u001b[0m - \u001b[1mget_current_motion_group_state took 0.10 seconds\u001b[0m\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "Pose(position=Vector3d(x=-91.40618174243164, y=-662.0208072831879, z=851.325747432342), orientation=Vector3d(x=2.1397304301105744, y=2.1396749657965426, z=-0.35713579892606034))"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "print(my_robot.tcps())\n",
-    "print([round(j,2) for j in my_robot.current_joints()])\n",
-    "print(my_robot.current_tcp_pose())"
+    "tcp_pose = await motion_group.tcp_pose(\"Flange\")\n",
```

Comment:
I would fetch the TCP in this example, so it right away works with any attached robot